### DISCUSSION TITLE
Acerca de fondo en el pokedex

### DISCUSSION THREAD

**Discussion by MarceloCortesC** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#540] [Q&A]
buenas tardes, dias o noches, disculpne alguien sabe como colocar una imagen de fopndo en el html sin que se repita, tenego el codigo de esta manera (adjunto imagen), coloco el body backgound="./Fondo.png", pero se repite,  intente colocarlo con style, asi style=\"background-image: url(/images/fondo1.jpg); background-repeat: no-repeat;\", pero tampoco tuve exito, alguien sabe de que manera puedo hacer eso.. por favor seria de mucha ayuda que me dijer
![sdf](https://user-images.githubusercontent.com/99509464/160259101-29006240-171d-4c17-b3ee-ad91ccc86806.JPG)
![asdhj](https://user-images.githubusercontent.com/99509464/160259102-b05f47d9-4b62-4367-b737-88a5ae2b77d7.JPG)
a , gracias por leer.

**Comment by AngelCruzO** [comment]
Haz una clase en css

.fondo{
background-image: url(/images/fondo1.jpg); 
background-repeat: no-repeat;
background-size: cover;
}

a la etiqueta body le pones class="fondo", asi debe de funcionar. Tambien checa bien la url de la imagen y si no recuerdo mal va entre "

**Reply by MarcosSilvaG** [reply]
efectiamente, la url va dentro de comillas simples o dobles :D y al parecer con lo que pones se le solucionará, esperemos responda xd

**Reply by AngelCruzO** [reply]
Esperemos que si, le funcione. Porque no veo otra forma para solucionarlo de manera simple
